src/main/java/io/botcrafting/botcraft/infra/controller/message/ErrorHandlingController.java

```diff
@@ -13,12 +13,11 @@
 @ControllerAdvice
 @Log
 public class ErrorHandlingController {
-	
 	  @ExceptionHandler(HttpClientErrorException.class)
 	  @ResponseStatus(HttpStatus.OK)
 	  @ResponseBody
-	  public ResponseEntity onHttpClientErrorException(HttpClientErrorException characterNameAlreadyExists) {
+	  public ResponseEntity<HttpClientErrorException> onHttpClientErrorException(HttpClientErrorException characterNameAlreadyExists) {
```

Comment:
I don't know if this is necessary, since the pointt here is to return http status 200 either way to avoid blocking posterior messages, so, using diamond operator here, for me, seems a little redundant. I gues that this ResponseEntity doesn't need to be explicitly typed. But IDK, let's hear @rbsilva  opnions on this one!

Reply:
I've changed to HttpStatus. 
It was made just to avoid warnings.